crates/primitives/starknet/src/execution/contract_class_wrapper.rs

```diff
@@ -152,6 +158,9 @@ impl From<Program> for ProgramWrapper {
             shared_program_data: value.shared_program_data,
             constants: value.constants,
             reference_manager: value.reference_manager,
+            // Defaulting to the latest compiler version.
+            compiler_version: "0.11.2".to_string(),
```

Comment:
should we create an issue to make it easily changeable in the future ? I am fine not doing it now but just so that we dont forget.

Reply:
yes